app/services/team_move_service.rb

```diff
@@ -34,6 +34,8 @@ def transfer_team
     @new_unit.name << " (Moved from #{@business_unit.directorate.name})"
     @new_unit.correspondence_type_roles = @business_unit.correspondence_type_roles
     @new_unit.properties = @business_unit.properties
+    @new_unit.user_roles = @business_unit.user_roles
+    @business_unit.user_roles.delete_all   
```

Comment:
Would prefer to replace `@business_unit.user_roles.delete_all` with a method such as `@business_unit.destroy_related_user_roles!` or similar to describe intent here

Reply:
Done